src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -249,6 +250,12 @@ class Microsoft::Terminal::Core::Terminal final :
         Down
     };
 
+    enum class SearchDirection

```

Comment:
Huh. Don't we already have an enum for this?

Reply:
```
src\buffer\out\search.h
27-
28-class Search final
29-{
30-public:
31:    enum class Direction
```

Whether it is usable here is a different story...

Reply:
Yeah, it's just not really worth it imo. I had a bool before but I think it's ok to have this enum be a bit redundant (at least for now).